data_interrogator/utils.py

```diff
@@ -1,17 +1,57 @@
 """A collection of useful functions that didn't really belong anywhere else"""
-from data_interrogator.interrogators import Allowable
-from django.conf import settings
+from enum import Enum
+from typing import Tuple, Union
 
 from django.apps import apps
+from django.conf import settings
 from django.db.models import Model
 
-from typing import Tuple, Union
+from data_interrogator.db import DateDiff, ForceDate
+
 import logging
 
 logger = logging.getLogger(__name__)
 logger.debug(f"Logging started for {__name__}")
 
 
+class Allowable(Enum):
+    ALL_APPS = 1
+    ALL_MODELS = 1
+    ALL_FIELDS = 3
+
+
+
+def normalise_field(text) -> str:
+    """Replace the UI access with the backend Django access"""
+    return text.strip().replace('(', '::').replace(')', '').replace(".", "__")
```

Comment:
Is replacing . with __ always necessary?

Reply:
Yes - django fields shouldn't include `__` dunderscores. If they do it will break the ORM anyway. And field names should never have '.'.
So, we can  safely replace `.` with `__`.